src/components/common/SendCode.vue

```diff
@@ -1,19 +1,31 @@
 <template>
   <div>
-    {{ $t('components.sendCode.didntGetCode') }}&nbsp;
-    <v-link
-      :disabled="isLoading"
-      href="#"
-      data-test="send-code"
-      @click.prevent="onClick"
-    >
-      {{ $t('components.sendCode.sendTitle') }}
-    </v-link>
+    <template v-if="isLocked">
```

Comment:
prefer to separate in two components and show they using `<component/>` as [example](https://github.com/endpass/endpass-auth/blob/master/src/components/forms/DocumentUploadForm/UploadFile/UploadFileFront/UploadFileFront.vue)